docs/getting-started.md

```diff
@@ -7,28 +7,126 @@ title: Getting Started
+Meld je aan bij het kadaster om [aan te sluiten en voor toegang tot de testomgeving](???????????????????????????). Je ontvangt dan o.a. een API-key die nodig is voor toegang tot de testomgeving.
 
 ## Functionaliteit en specificaties
 Je kunt een visuele representatie van de specificatie bekijken met [Swagger UI]({{ site.baseurl }}/swagger-ui) of [Redoc]({{ site.baseurl }}/redoc).
 
-* Je kunt een visuele representatie van de specificatie bekijken met [Swagger UI](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/swagger-ui) en in [Redoc](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/redoc)
-* Je kunt de [Technische specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/tree/master/specificatie/genereervariant/openapi.yaml) downloaden
-* Je kunt de [functionele documentatie](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/features) vinden in [features](https://vng-realisatie.github.io/Haal-Centraal-WOZ-bevragen/features).
+Je kunt de [functionele documentatie](./features) vinden in de [features](./features).
+
+### Resource WOZ-object 
+Je kunt op de volgende manieren WOZ objecten (met WOZ waardes) zoeken en raadplegen:
+
+- Opvragen van 1 specifiek WOZ-object met een identificatie van het WOZ object.
+- Opvragen van 1 specifiek WOZ-object met de identificatie van het adres.
+- Opvragen van 1 specifiek WOZ-object met de postcode en het huisnummer eventueel aangevuld met huisletter en/of huisnummertoevoeging.
+- Opvragen van collectie WOZ-objecten met een niet-natuurlijk persoon (rsin) of een maatschappelijke activiteit (kvknr) die in het Handelsregister is ingeschreven.
+- Opvragen van collectie WOZ-objecten in eigendom van een bij het Handelsregister ingeschreven maatschappelijke activiteit of een van de daaronder vallende ondernemingen en vestigingen.
+- Opvragen van collectie WOZ-objecten met de identificatie van een adresseerbaar object (verblijfsobject, ligplaats, standplaats).
+
+### Algemeen
+Verder zijn er nog een paar algemene functies die gelden voor alle bovenstaande aanvragen:
+- Gebruik van de fields parameter om de response te filteren. (Voor werking, zie feature [fields](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/fields.feature))
+- Velden die altijd worden geleverd.
+
+|Resource                           |Velden                         |
+|-----                              |------                         |
+|wozobjecten                        |identificatie, _links.self     |
+
+- [HAL links](https://tools.ietf.org/html/draft-kelly-json-hal-08), die soms [templated](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/uri-templating.feature) worden geleverd.
 
 ## Bouw de API
 We hebben [client code](https://github.com/VNG-Realisatie/Haal-Centraal-WOZ-bevragen/tree/master/code){:target="_blank" rel="noopener"} voor API-clients in enkele varianten. Hiermee kan je direct aan de slag met het gebruiken van de API.
```

Comment:
de client code is al een tijd niet vernieuwd. wordt dat nog in gang gezet?

Reply:
is bij deze vernieuwd.